pkg/agent/multicast/mcast_discovery.go

```diff
@@ -130,20 +130,23 @@ func (s *IGMPSnooper) validate(event *mcastGroupEvent, igmpType uint8, packetInD
 	if event.iface.Type != interfacestore.ContainerInterface {
 		return true, fmt.Errorf("interface is not container")
 	}
-	// Validate checks if packet should be dropped or not, and returns multicast NP information
-	item, err := s.validator.Validate(event.iface.PodName, event.iface.PodNamespace, event.group, igmpType)
+	// GetIGMPNPRuleInfo looks up the IGMP NetworkPolicy rule that matches the given Pod and groupAddress,
```

Comment:
I feel we can remove this comment as it just repeats the same on the ClusterInfoImport() func.

Reply:
removed